### PR TITLE
ZUR-164 Add the modal for the channels description

### PR DESCRIPTION
**Pull request by TTUmejiaku**
Implemented the modal for the channel's description to work on desktop and mobile.
chatroom-modal-mobile
chatroom-modal

### CODE REVIEW

**Comment by JulianaSau** [comment]
Why is there a red border around the close button?
Also send the link to the design you're following here @TTUmejiaku

File: packages/ui/src/message-room-view-header/MessageRoomViewHeader.jsx [hunk 1]
```diff
@@ -22,10 +22,12 @@ export default function MessageRoomViewHeader(props) {
       <div className={styles.plugin__header}>
         <div
           onClick={() => {
-            if (props.roomInfo) {
```
Comment (JulianaSau) [inline packages/ui/src/message-room-view-header/MessageRoomViewHeader.jsx:25]: Why did you remove the if block? Is the room info not being received on the frontend?

**Comment by TTUmejiaku** [comment]
I asked for the design but El'Tana told me that I should just imitate what's on slack...so I didn't follow any design

File: packages/ui/src/message-room-view-header/MessageRoomViewHeader.jsx (hunk 1 above)
Comment (TTUmejiaku) [reply packages/ui/src/message-room-view-header/MessageRoomViewHeader.jsx:25]: Yes, it's not...it's returning undefined

**Comment by TTUmejiaku** [comment]
No specific reason for the red border...should I remove it then?

**Comment by JulianaSau** [comment]
> No specific reason for the red border...should I remove it then?

Yes. It's not needed there

File: packages/ui/src/message-room-view-header/MessageRoomViewHeader.jsx (hunk 1 above)
Comment (JulianaSau) [reply packages/ui/src/message-room-view-header/MessageRoomViewHeader.jsx:25]: Okay. I'll look into that

**Comment by TTUmejiaku** [comment]
I checked the code this morning, and everything looks fine...this is the code that controls the button👇
` <Button onClick={close}>
                  <AiOutlineClose size="20px" color="gray" />
                </Button>
`

chatroom-modal-mobile
👇

**Comment by JulianaSau** [comment]
Hi @TTUmejiaku, my issue with it is the border around the close icon. That should only appear if the button is in focus or has been pressed. it should'nt be there in the current state

**Comment by TTUmejiaku** [comment]
@JulianaSau, the border is part of the icon, except I have to change it?

**Comment by TTUmejiaku** [comment]
@JulianaSau I've affected the changes. Implemented the border to appear on hover because you won't be able to see it on pressed because the modal will close.
chatroom-modal
chatroom-modal-mobile

**Comment by JulianaSau** [comment]
Pull changes from dev branch so that i can merge

**Comment by TTUmejiaku** [comment]
> Pull changes from dev branch so that i can merge

done

**Comment by JulianaSau** [comment]
Pull again @TTUmejiaku. I just merged another pr
